source/Outfit.cpp

```diff
@@ -471,6 +479,20 @@ const Dictionary &Outfit::Attributes() const
 
 
 
+const Weapon &Outfit::GetWeapon() const
+{
+	return weapon;
+}
+
+
+
+const Outfit *Outfit::Ammo() const
+{
+	return weapon.Ammo();
+}
```

Comment:
I don't think we should be implementing functions like that, that just call the same function as the weapon, because people can simply use GetWeapon().Ammo() if they need it.

Reply:
removed it